todo/tests/test_views.py

```diff
@@ -133,6 +133,64 @@ def test_no_javascript_in_task_note(todo_setup, client):
     assert task.note == bleach.clean(note, strip=True)
 
 
+@pytest.mark.django_db
+def test_completed_unchanged(todo_setup, client):
+
+    def makeTask(title, client):
+        task_list = TaskList.objects.first()
+        u2 = get_user_model().objects.get(username="u2")
+        
+        data = {
+            "task_list": task_list.id,
+            "created_by": u2.id,
+            "priority": 10,
+            "title": title,
+            "note": "a note",
+            "add_edit_task": "Submit",
+        }
+
+        url_add_task = reverse("todo:list_detail", kwargs={"list_id": task_list.id, "list_slug": task_list.slug})
+
+        response = client.post(url_add_task, data)
+        assert response.status_code == 302
+
+    title = "Some Unique String with unique chars: xpf893sgyqrzx"
+    client.login(username="u2", password="password")
+    makeTask(title, client)
```

Comment:
In  pytest we don't usually call functions manually to generate fixture data - we pass in external functions as fixtures (see the other tests in `test_views`).